### PR TITLE
Add a bio for @alcala21

### PR DESCRIPTION
**Pull request by alcala21**
This Pull Request adds a bio for @alcala21 as requested in #16. 

@alcala21, what do you think of these changes?

### CODE REVIEW

File: docs/alcala21.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# alcala21
+
+**Name:** Carlos F. Alcala
+
+**Ocupation:** Research Engineer at Johnson Controls
+
+**Location:** Milwaukee, Wisconsin
```
Comment (alcala21) [inline docs/alcala21.md]: Wow. Milwaukee is a cool city. :city_sunrise:

**Comment by alcala21** [comment]
Looks good!

File: docs/alcala21.md (hunk 1 above)
Comment (ghost) [inline docs/alcala21.md]: maybe you should add something like 507 E Michigan Facility or something

**Comment by alcala21** [comment]
@ryanjames28, how about now?

**Comment by alcala21** [comment]
:+1:

:shipit:

**Pull request merged**